vignettes/vignette_best_practices.Rmd

```diff
@@ -0,0 +1,198 @@
+---
+title: "Best practices when writing vignette in admiral"
+output: rmarkdown::html_vignette
```

Comment:
Vignesh indicated to me at one point that when creating vignettes, the option `toc:false` should be specified.

Reply:
I think we discussed it and agreed we would keep the default. I have checked both a vignette with `toc:false` (bds) and one with the default (imputation dates) and I could not see any difference in the display of the ToC.
Did you recall why Vignesh recommended this?

Reply:
@kabis-ops I reached out to an R expert at GSK on this.  Her thoughts were that it is not a requirement to have `toc: false` but if you have a vignette that requires a TOC then it should probably be broken down into multiple vignettes as a best practice.  You can close this comment or consider including the `toc: false` as a best practice in admiral.